docs/guides/azure-private-link-workspace-simplified.md

```diff
@@ -0,0 +1,310 @@
+---
+page_title: "Provisioning Databricks on Azure with PrivateLink - Simplified deployment"
```

Comment:
if you run `make fmt-docs`, it will format all the Terraform snippets in the doc